inst/qml/PenalizedMetaAnalysis.qml

```diff
@@ -0,0 +1,415 @@
+		*/
+	}
+
+	Section
+	{
+		title: qsTr("Priors")
+
+		Group
+		{
+			title:		qsTr("Horseshoe")
+			visible:	method.currentValue == "hs"
+			
+			DoubleField
+			{
+				name:			"priorHsDf"
+				label:			qsTr("Df")
+				defaultValue:	1
+				min:			1
+				inclusive:		JASP.MinOnly
+			}
+
+			DoubleField
+			{
+				name:			"priorHsScale"
+				label:			qsTr("Scale")
+				defaultValue:	1
+				min:			0
+				inclusive:		JASP.None
+			}
+		}
+
+		Group
+		{
+			title:		qsTr("Lasso")
+			visible:	method.currentValue == "lasso"
+			
+			DoubleField
+			{
+				name:			"priorLassoDf"
+				label:			qsTr("Df")
+				defaultValue:	1
+				min:			1
+				inclusive:		JASP.MinOnly
+			}
+
+			DoubleField
+			{
+				name:			"priorLassoDfGlobal"
+				label:			qsTr("Df (global)")
+				defaultValue:	1
+				min:			1
+				inclusive:		JASP.MinOnly
+			}
+
+			DoubleField
+			{
+				name:			"priorLassoDfSlab"
+				label:			qsTr("Df (slab)")
+				defaultValue:	4
+				min:			1
+				inclusive:		JASP.MinOnly
+			}
+
+			DoubleField
+			{
+				name:			"priorLassoScaleGlobal"
+				label:			qsTr("Scale (global)")
+				defaultValue:	1
+				min:			0
+				inclusive:		JASP.None
+			}
+
+			DoubleField
+			{
+				name:			"priorLassoScaleSlab"
+				label:			qsTr("Scale (slab)")
+				defaultValue:	1
+				min:			0
+				inclusive:		JASP.None
+			}
+		}
+	}
+
+	Section
+	{
+		title:		qsTr("Inference")
+		columns:	1
+
+		Group
+		{
+			CheckBox
+			{
+				name:		"estimatesCoefficients"
+				label:		qsTr("Estimates")
+				checked:	true
+			}
+
+			CheckBox
+			{
+				name:		"estimatesTau"
+				label:		qsTr("Heterogeneity")
+				checked:	false
+
+				CheckBox
+				{
+					name:		"estimatesI2"
+					label:		qsTr("I²")
+					checked:	false
+				}
+			}
+		}
+
+		Group
+		{
+			title:				qsTr("Posterior plots")
+			
+			VariablesForm
+			{
+				preferredHeight:	250 * preferencesModel.uiScale
+				id:					availableModelComponentsForms
+				property var alwaysAvailable:
+					[
+						{ label:	"Intercept",		value: "Intercept"},
+						{ label:	"Heterogeneity",	value: "Heterogeneity"}
+					]
+
+				AvailableVariablesList
+				{
+					name:	"availableModelComponentsPlot"
+					title:	qsTr("Model terms")
+					source:	["modelTerms", {values: availableModelComponentsForms.alwaysAvailable}]
+				}
+
+				AssignedVariablesList
+				{
+					name:	"plotPosterior"
+					title:	qsTr("Plotted term")
+				}
+			}
+		}
+
+	}
+
+
+	Section
+	{
+		title:		qsTr("MCMC diagnostics")
+
+		VariablesForm
+		{
+			preferredHeight: 200 * preferencesModel.uiScale
+			id:				diagnosticsForms
+			property var alwaysAvailable:
+				[
+					{ label:	"Intercept",		value: "Intercept"},
+					{ label:	"Heterogeneity",	value: "Heterogeneity"}
+				]
+
+			AvailableVariablesList
+			{
+				name:	"diagnosticsVariables"
+				title:	qsTr("Model terms")
+				
+				source:	["modelTerms", {values: diagnosticsForms.alwaysAvailable}]
+			}
+
+			AssignedVariablesList
+			{
+				singleVariable:	true
+				name:			"diagnosticsVariable1"
```

Comment:
use some more descriptive name...

Reply:
also for `diagnosticsVariable2`

Reply:
changed to `scatterVariableX` and `scatterVariableY`